docs/ingestion/native-batch.md

```diff
@@ -345,10 +345,22 @@ The Parallel task will use one subtask when you set `maxNumConcurrentSubTasks` t
 
 When you use this technique to partition your data, segment sizes may be unequally distributed if the data in your `partitionDimension` is also unequally distributed.  Therefore, to avoid imbalance in data layout, review the distribution of values in your source data before deciding on a partitioning strategy.
 
-For segment pruning to be effective and translate into better query performance, you must use
-the `partitionDimension` at query time.  You can concatenate values from multiple
-dimensions into a new dimension to use as the `partitionDimension`. In this case, you
-must use that new dimension in your native filter `WHERE` clause.
+Range partitioning has several benefits:
+
+1. Lower storage footprint due to combining similar data into the same segments, which improves compressibility.
+2. Better query performance due to Broker-level segment pruning, which removes segments from
+   consideration when they cannot possibly contain data matching the query filter.
+
+For Broker-level segment pruning to be effective, you must include the `partitionDimension` in the
+`WHERE` clause at query time, using a filter that supports pruning. Filters that support pruning
+include:
+
+- Equality, like `x = 'foo'` or `x IN ('foo', 'bar')`
+- Comparison on strings, like `x < 'foo'` or other comparisons involving `<`, `>`, `<=`, or `>=`.
+  Comparisons on numbers do not support pruning.
```

Comment:
Does this mean Equality on numbers like `x = 123` supports segment pruning?

Reply:
Good question. It depends on the type of `x`. I updated the language to be clearer.